geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractHitsMissesIntegrationTest.java

```diff
@@ -65,8 +65,8 @@ public void teardown() {
   /***********************************************
    ************* Supported Commands **************
    **********************************************/
```

Comment:
Do you think we should eliminate the Supported/Unsupported sections altogether?

Reply:
for what reason?

Reply:
Since the goal is eventually to get rid of Supported/Unsupported, it seems unnecessary to maintain that distinction here.  I recognize we still have that distinction in `RedisCommandType` and the Supported/Unsupported JUnit test, but that is to make sure a user is not able to access those commands.  I don't really see a benefit to maintaining that distinction here.  But maybe since it's already like that we could just move the commands out from the `Unsupported` section over time.